charts/wandb/templates/role.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: secret-manager-role
```

Comment:
why is this static

Reply:
This can be whatever pattern. Whatever name you want.

Reply:
Everything should be based off the chart name

Reply:
Done.